components/soc/esp32/rtc_clk_init.c

```diff
@@ -119,6 +119,7 @@ void rtc_clk_init(rtc_clk_config_t cfg)
 
     bool res = rtc_clk_cpu_freq_mhz_to_config(cfg.cpu_freq_mhz, &new_config);
     assert(res && "invalid CPU frequency value");
+    rtc_clk_cpu_freq_set_config_fast(&new_config);
```

Comment:
Thanks! This should probably be non-fast version, since _fast can't enable PLL, if needed?

Reply:
I thought _fast has fallback to full version if required?

Reply:
It does, but that relies on static variables which are not set at that point yet